src/components/ArticleHolder.jsx

```diff
@@ -0,0 +1,49 @@
+// import axios from 'axios';
+import {useState, useEffect} from 'react';
+// import { Routes, Route } from "react-router-dom";
+// import Article from './Article';
+
+
+
+function ArticleHolder () {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        fetch(
+          `https://nc-news-by-victor.onrender.com/api/articles`
```

Comment:
Could we try and refactor this to axios and extract to it's own api.js file when we get chance please?